lib/Raven/ClientBuilder.php

```diff
@@ -201,6 +206,40 @@ public function removeHttpClientPlugin($className)
         return $this;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function addMiddleware(callable $middleware, $priority = 0)
+    {
+        $this->middlewares[] = [$middleware, $priority];
```

Comment:
Shouldn't we sort them on insertion as an optimization?
Also that would mean that the getter would serve them always sorted...

Reply:
In the case of the client builder the getter is there just to return the list of middlewares that will be added to the client and their priority, so sorting the array is useless as it's just a temporary place where the data is stored